src/lib/constants.ts

```diff
@@ -59,6 +59,12 @@ export const KCL_DEFAULT_CONSTANT_PREFIXES = {
 /** The default KCL length expression */
 export const KCL_DEFAULT_LENGTH = `5`
 
+/** The default KCL tolerance expression */
+export const KCL_DEFAULT_TOLERANCE = `0.1mm`
```

Comment:
Wondering if this is to make sure people use absolute units there, even if they work in, say, inches?

Reply:
Ok thanks @jamwaffles, you were right @max-mrgrsk !